packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -99,9 +99,19 @@ module.exports = {
           additionalProperties: false,
           required: ['subnet'],
         },
+        stop: {
```

Comment:
I think, we should add this to every service's docker settings since stop timeout highly depends on service graceful shutdown logic.